src/import/sst_service.rs

```diff
@@ -618,6 +623,74 @@ impl<E: Engine> ImportSstService<E> {
     }
 }
 
+fn check_local_region_stale(
+    region_id: u64,
+    epoch: &RegionEpoch,
+    region_info_accessor: Arc<dyn RegionInfoProvider>,
+) -> Result<()> {
+    let (tx2, rx2) = crossbeam::channel::bounded(1);
```

Comment:
You can use `paired_future_callback` to make the function async, it can improve performance I think.